src/webrtc/connection.ts

```diff
@@ -298,10 +303,13 @@ class WebRTCConnection implements MultiaddrConnection {
         )
       )
     )
+    this.verbose(`FLOW: sending UPGRADED to relay`)
+    this.relayConn.sendUpgraded()
```

Comment:
What happens if WebRTC is unavailabe?

Reply:
Right, I forgot this small fix. Pushed now. Thx.